winui/DataGrid/Export-To-Excel.md

```diff
@@ -0,0 +1,653 @@
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting merged cells
+
+You can export merged cells to excel by setting `CanExportMergedCells` property as `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportMergedCells = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Exporting unbound rows
+
+You can export unbound rows to excel by setting `CanExportUnboundRows` property as `true`.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.CanExportUnboundRows = true;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+### Changing start row and column index while exporting
+
+You can export the data to specified row index and column index in worksheet, by setting `StartRowIndex` and `StartColumnIndex` properties.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+options.StartColumnIndex = 3;
+options.StartRowIndex = 3;
+var excelEngine = dataGrid.ExportToExcel(dataGrid.View, options);
+var workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-with-row-column-index.png" alt="Exporting to Excel with Custom Row and Column Index in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Export DataGrid SelectedItems to Excel
+
+By default, entire grid will be exported to Excel. You can export selected rows only by passing `SelectedItems` to 
+`ExportToExcel` method.
+
+{% tabs %}
+{% highlight c# %}
+var options = new DataGridExcelExportOptions();
+var excelEngine = dataGrid.ExportToExcel(dataGrid.SelectedItems, options);
+IWorkbook workBook = excelEngine.Excel.Workbooks[0];
+MemoryStream stream = new MemoryStream();
+workBook.SaveAs(stream);
+Save(stream, "Sample");
+{% endhighlight %}
+{% endtabs %}
+
+<img src="Export-to-Excel_images/winui-datagrid-export-selected-item.png" alt="Exporting Selected Items only to Excel in WinUI DataGrid" width="100%" Height="Auto"/>
+
+## Row Height and Column Width customization 
+
+After exporting data to excel, you can set different row height and column width for the columns based on your requirement. You can refer [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information. 
```

Comment:
After exporting data to excel, set the different row heights and column widths for the columns based on your requirement. Please refer [here](http://help.syncfusion.com/file-formats/xlsio/worksheet-rows-and-columns-manipulation#adjust-row-height-and-column-width) for more information.